docs/proposals/tenant-deletion.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "Deletion of Tenant Data from Blocks Storage"
```

Comment:
I'm guessing you didn't mean to have this file in the review since it looks unrelated?

Reply:
Oh, right. Thanks. That is part of different PR.